backend/core/package.json

```diff
@@ -126,4 +126,4 @@
     "node": "12",
     "yarn": "^1.22"
   }
-}
```

Comment:
Do you have an editor that removes the newline at the end of JSON files?

Reply:
Haha what was your first clue? ;)

Fixed it! (Both my VSCode settings, and re-added the newlines)